### DISCUSSION TITLE
3.5inch-dark improved

### DISCUSSION THREAD

**Discussion by the-unsoul** [mathoudebine/turing-smart-screen-python#209] [Themes]
A small improvement over @Pepper-the-kobold [shared dark background](https://github.com/mathoudebine/turing-smart-screen-python/discussions/58#discussioncomment-4940788)

![preview](https://user-images.githubusercontent.com/9525030/235353032-09e47dba-04a7-48fe-9564-81e12a04725a.png)

Changes:

- Shorten hard drive usage to `{used} / {total} G`
- Moves date-time to hard-drive section & increased font size
- Slightly adjust bars color
- Adjusted text color for some outputs

Download:
[3.5inch-dark.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/11361307/3.5inch-dark.zip)

Install:
Extract archive into res/themes
Runs `python configure.py` then select `3.5inch-dark`

**Comment by Pepper-the-kobold** [comment]
Oooh, I like it. I'll be using this myself. Haha!

**Comment by Paradoxum** [comment]
Thanks :)
